### ISSUE TITLE
Accordion has WCAG 4.1.2 nested-interactive errors

### ISSUE THREAD

**Issue by EricBPower** [valor-software/ng2-bootstrap#6203]
Before opening an issue, please search for duplicates (opened and closed) https://github.com/valor-software/ngx-bootstrap/issues There's no need to open an issue here if you want to ask general question, use StackOverflow or Slack instead

Bug description:  The ngx-bootstrap implementation of accordions has WCAG 4.1.2 errors reported for it. You can see the errors using your ngx-bootstrap [accordion docs/demo page](https://valor-software.com/ngx-bootstrap/old/6.2.0/#/accordion) using an axe wcag inspection tool which runs on axe version 4.2.
We also get this on our own site using ngx-bootstrap accordions, but while on your demo page you can use this chrome extension to verify WCAG compatibility:  https://chrome.google.com/webstore/detail/accessibility-insights-fo/pbjjkligggfmakdaogkfomddhfmpjeni

You will see all uses of the accordion get a WCAG 4.1.2 error for "nested-interactive: Nested interactive controls are not announced by screen readers".   This is unique to your implementation, the [regular Bootstrap website demo](https://getbootstrap.com/docs/4.3/components/collapse/#accordion-example) does not have this issue for it's demo of accordion.

Versions of ngx-bootstrap, Angular, and Bootstrap:

axe: 4.2
ngx-bootstrap: 6.2.0
Angular: 11
Bootstrap: 4.3

Build system: Angular CLI

**Expected behavior**

Accordion component should pass the nested-interactive WCAG rules.

Title: WCAG 4.1.2: Nested interactive controls are not announced by screen readers (.accordion-toggle[role="button"][_ngcontent-wet-c61=""])
Tags: Accessibility, WCAG 4.1.2, nested-interactive

Issue: Nested interactive controls are not announced by screen readers (nested-interactive - https://dequeuniversity.com/rules/axe/4.2/nested-interactive?application=msftAI)

Target application: Angular Bootstrap - https://valor-software.com/ngx-bootstrap/accordion

Element path: demo-accordion-basic > accordion[aria-multiselectable="false"] > accordion-group[heading="Static\ Header"] > .panel-default.card.panel > .panel-heading.card-header.panel-enabled > .panel-title[_ngcontent-wet-c61=""] > .accordion-toggle[role="button"][_ngcontent-wet-c61=""]

Snippet: <div _ngcontent-wet-c61="" role="button" class="accordion-toggle" aria-expanded="false"><button _ngcontent-wet-c61="" type="button" class="btn btn-link"> Static Header </button></div>

How to fix: 
Fix any of the following:
  Element has focusable descendants

Environment: Chrome version 91.0.4472.124

====

This accessibility issue was found using Accessibility Insights for Web 2.27.0 (axe-core 4.2.1), a tool that helps find and fix accessibility issues. Get more information & download this tool at http://aka.ms/AccessibilityInsights.
